hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -30,6 +30,16 @@ cpu_rst = top["reset_paths"]["sys"]
 dm_rst = top["reset_paths"]["lc"]
 %>\
 module top_${top["name"]} #(
+  // Auto-inferred parameters
+% for m in top["module"]:
+  % if len(m["param_list"]) != 0:
```

Comment:
I don't think you need this if statement (because iterating over an empty list doesn't do anything).

Reply:
I wasn't aware of that, changed.